src/liger_kernel/transformers/layer_norm.py

```diff
@@ -23,3 +25,6 @@ def forward(self, hidden_states):
         return LigerLayerNormFunction.apply(
             hidden_states, self.weight, self.bias, self.variance_epsilon
         )
+        
+    def extra_repr(self) -> str:
+        return f'{self.hidden_size}, eps={self.eps}'
```

Comment:
why no weight here?

Reply:
This is in line with nn.LayerNorm whose extra_repr is:
```python
def extra_repr(self) -> str:
        return '{normalized_shape}, eps={eps}, ' \
            'elementwise_affine={elementwise_affine}'.format(**self.__dict__)
```
For rms_norm, it is in line with huggingface transformers.

Reply:
Do we need to keep them consistent here?

Reply:
oh sg